cmds/tui.c

```diff
@@ -2028,6 +2004,50 @@ static bool tui_window_expand(struct tui_window *win)
 	return win->ops->expand(win, win->curr, 1);
 }
 
+/* move to the previous sibling */
+static bool tui_window_move_prev(struct tui_window *win)
+{
+	void *prev = win->ops->sibling_prev(win, win->curr);
+	int count = 0;
+
+	if (prev == NULL)
+		return false;
+
+	/* fold the current node before moving to the previous sibling */
+	if (win->ops->collapse)
+		count = win->ops->collapse(win, win->curr, 0);
+
+	while (win->curr != prev)
+		tui_window_move_up(win);
+
+	/* collapse the current node after moving to the previous sibling */
+	count += tui_window_collapse(win);
```

Comment:
Instead of moving the code (in order to use `tui_window_collapse` I guess), you can call `win->ops->collapse(depth = 1)` directly.  I believe it's better to understand what it does actually.

Reply:
Okay. I will do that. Thanks!